doc/decoding/decoding_intro.rst

```diff
@@ -463,13 +463,13 @@ We can try Fisher's `Linear Discriminant Analysis (LDA)
 
 Import the module::
 
-    >>> from sklearn.lda import LDA
+    >>> from sklearn.lda import LDA  # doctest: +SKIP
```

Comment:
I think that it would be better to do it the other way around: change the code to import from `discriminant_analysis`, and add a note saying that in old versions of scikit-learn, it was imported from lda. Indeed, change in sklearn has occurred in the 0.17 release, which was in November 2015, so that's quite a while ago.